src/components/Home/TransparencyPersonalDataSection.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * TransparencyPersonalDataSection is a component that represents
+ * the section of the Homepage page introducing the overview features
+ */
+
+// img
+import OverviewPic from '../../assets/overview-page.png';
+
+//utils
+import styled from 'styled-components';
+
+// MUI components
+import { Box, Typography } from '@mui/material';
+
+// Typescript types
+interface ExplanationList {
+  content: string;
+}
+
+// Contains each content explanation paragraph
+const explanationList: ExplanationList[] = [
+  {
+    content: `Avastar provides you with a dashboard to visualise which companies are tracking you, which types of personal data are collected and informs you on how they can be used. `,
+  },
+  {
+    content: `We automated the whole process to visualise the raw personal data you requested to platforms so you can focus on making decisions about how you manage your online privacy.
+    `,
+  },
+  {
+    content: `With Avastar, you are only a few clicks away from having a full glance of your digital identity!
+    `,
+  },
+];
+
+// Styled-components
+const StyledExplanation = styled.section`
+  padding: 0 9.438rem 5.688rem 10.063rem;
+  width:100%;
+  display:flex;
+  flex-direction:column;
+  align-items:center;
+`;
+const ImgContainer = styled(Box)`
+  width: 100%;
+`;
+
+const Img = styled.img`
+  width: 31.438rem;
+  height: 21.938rem;
+`;
+
+const Title = styled(Typography)`
+  padding-bottom: 2.75rem;
+`;
+const ParagraphList = styled.ul`
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  gap: 1.25rem;
+`;
+
+const ParagraphItem = styled.li``;
+
+export default function TransparencyOverview() {
+  return (
+    <StyledExplanation>
+        <Title id="services"
+          sx={{
+            lineHeight: '2.5rem',
+            fontWeight: 700,
+            fontSize: '2.074rem',
+          }}
+        >
+          Get back the transparency you deserve on your personal data !
```

Comment:
Suite à des discussion sur Figma, on s'est alignés sur le retrait du mot "back" pour avoir juste "Get the transparency you deserver on your personal data !"

Reply:
Exact, c'est modifié :)